client/index.html

```diff
@@ -30,10 +32,44 @@ <h1> Sign In</h1>
 			</form>
 			
 		</div>
+
+		
+		<nav class="navbar navbar-default navbar-expand navbar-dark bg-dark">
+				<a href="#" class="navbar-brand">Torchbearer</a>
+				<button class="navbar-toggler" data-toggle="collapse" data-target="#navbarMenu">
+					<span class="navbar-toggler-icon"></span>
+			</button>
+			<div class="collapse navbar-collapse" id="navbarMenu">
+				<ul class="navbar-nav">
+					<li class="nav-item">
+						<a href="#" class="nav-link">Home</a>
+					</li>
+					<li class="nav-item">
+						<a href="#" class="nav-link">Board</a>
+					</li>
+					<li class="nav-item">
+						<a href="#" class="nav-link">Tasks</a>
+					</li>
+
+				</ul>
+
+			</div>
+
+
+		</nav> 
+
+
+
 	
 	<!-- ---------------------------------------- Bootstrap ---------------------------------------- -->
 	<script src="https://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
 	<script src = "https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.bundle.min.js"></script>
+	
```

Comment:
This section has two references to jquery and bootstrap. Not an error just redundant.